examples/tensorflow/tf2/train_predict_2.py

```diff
@@ -8,7 +8,6 @@
 import tempfile
 import tensorflow as tf
 from tensorflow import estimator as tf_estimator
-import mlflow.tensorflow
```

Comment:
This example calls `mlflow.tensorflow.autolog()` - I think it will fail if we remove this import. Perhaps we can remove this example for now and file a follow-up ticket to introduce a TF Core example?

Reply:
Filed ticket https://databricks.atlassian.net/browse/ML-25289